scripts/release/homebrew/docker/formula_template.txt

```diff
@@ -9,9 +9,15 @@ class AzureCli < Formula
   license "MIT"
   head "https://github.com/Azure/azure-cli.git"
 
+  livecheck do
+    url "https://github.com/Azure/azure-cli/releases/latest"
+    regex(%r{href=.*?/tag/azure-cli[._-]v?(\d+(?:\.\d+)+)["' >]}i)
```

Comment:
Can we add some comments about what this regex does?

Reply:
It's added by Homebrew team, seems to parsing the version number from `https://github.com/Azure/azure-cli/releases/latest` and then do [livecheck](https://github.com/Homebrew/homebrew-livecheck).